server/api.js

```diff
@@ -119,14 +121,16 @@ router.post("/graduates", function (req, res) {
 
 
 
-//checking the github username exist in our database
+//checking the github username exist in our database and if he is admin
 router.get("/accounts/:name", (req, res) => {
   const githubName = req.params.name;
   Connection.query(
     "SELECT * FROM github_accounts where account_name=$1 ",
     [githubName],
     (error, result) => {
-      if (result && (result.rowCount > 0)) {
+      if (result && (result.rowCount > 0)&& result.rows[0].is_admin) {
```

Comment:
this whole block is very convoluted and difficult to understand, would be useful to clean it up a little